README.md

```diff
@@ -1374,13 +1374,14 @@ _You can enable the following settings in Xcode by running [this script](resourc
   * `// MARK: Fileprivate` for `fileprivate` properties and methods.
   * `// MARK: Private` for `private` properties and methods.
   * If the type in question is an enum, its cases should go above the first `// MARK:`.
-  * If there are typealiases, they should go above the first `// MARK:`.
+  * If there are typealiases or nested types, they should go above the first `// MARK:`.
   * Do not subdivide each of these sections into subsections, as it makes the method dropdown more cluttered and therefore less useful. Instead, group methods by functionality and use smart naming to make clear which methods are related. If there are enough methods that sub-sections seem necessary, consider refactoring your code into multiple types.
-  * If the type in question is a simple value type, it is OK to omit the `// MARK:`s, as it would hurt legibility.
+  * If all of a type's definitions belong to the same category (e.g. the type only consists of `internal` properties), it is OK to omit the `// MARK:`s.
```

Comment:
This is how the `organizeDeclarations` rule behaves, so I think we should capture it in the rule description.

Reply:
I'm OK with this.